File: criticalpath_angular/src/app/model/storeCloseDateRequest.ts
export class StoreCloseDateRequest {
    
     public name : String;
    
     public description : String;
    
     public closeDate : String;
    
	 public stores : any[] = [];

	 public isActive : boolean;
    
     constructor(data : any) {
        
        this.name = data.name;
        this.description = data.description;
        this.closeDate = data.closeDate;
        this.stores = data.stores;
        this.isActive = data.isActive;
    }
}

File: criticalpath_server/src/main/java/com/ackermans/criticalpath/admin/dto/StoreCloseDateRequest.java
package com.ackermans.criticalpath.admin.dto;

import java.util.Set;

import com.ackermans.criticalpath.entity.Store;

import lombok.Data;

@Data
public class StoreCloseDateRequest {

	private Long id;
	
	private String name;
	
	private String description;
	
	private String closeDate;
	
	private Boolean isActive;
	
	private Set<Store> stores;
}
